source/2_platform/rex_windows/src/gui_application.cpp

```diff
@@ -142,6 +187,101 @@ namespace rex
         }
       }
 
+      void start_resize()
```

Comment:
maybe best to specify the "on_" prefix for all callbacks for events.
makes them easier to distinquish from other functions